lib/rubocop/cop/rspec/http_status.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'symbol_to_status_code'
+
+module RuboCop
+  module Cop
+    module RSpec
+      # This cop enforces consistent use of `symbol` or `numeric`
```

Comment:
You need a single line first-sentense description. You may add more on the next line